src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Color.values().length);
```

Comment:
You need to generate a new index for each method invocation - you currently generate it only once, when the class is created, and then getRandomColor returns that color each time the method is called. Make it a local variable instead